# Goal
bool lift_and_lock
---
# Result
bool success
---
# Feedback
